WordPress/Classes/Services/PageLayoutService.swift

```diff
@@ -82,7 +82,7 @@ extension PageLayoutService {
     static func resultsController(forBlog blog: Blog, delegate: NSFetchedResultsControllerDelegate? = nil) -> NSFetchedResultsController<PageTemplateCategory> {
         let context = ContextManager.shared.mainContext
         let request: NSFetchRequest<PageTemplateCategory> = PageTemplateCategory.fetchRequest(forBlog: blog)
-        let sort = NSSortDescriptor(key: "title", ascending: true)
+        let sort = NSSortDescriptor(key: "ordinal", ascending: true)
```

Comment:
Not a change request: To share a neat trick I recently came across you can leverage the name of the value we're sorting on by:
```suggestion
        let sort = NSSortDescriptor(key: #keyPath(PageTemplateCategory.ordinal), ascending: true)
```

Reply:
Thanks Chip! I really like this (it kinda bothered me that this was just a string literal). :+1: